internal/controller/api_to_config.go

```diff
@@ -514,6 +514,18 @@ func localPrefPrefixesToMap(withLocalPref []v1beta1.LocalPrefPrefixes) (localPre
 		localPrefForPrefixV6: map[string]uint32{},
 	}
 
+	// error when input is
+	// Prefixes:  []string{"192.0.2.0/24"},LocalPref: 100},
```

Comment:
let's make the case we are addressing clearer here: withLocalPref = []v1beta1.LocalPrefPrefixes{...}